infrastructure/main.tf

```diff
@@ -107,6 +107,7 @@ module "org-role-mapping-database-v15" {
   component          = var.component
   common_tags        = var.common_tags
   pgsql_version      = "15"
+  pgsql_sku          = var.pgsql_sku
```

Comment:
i think it’s because the inner branches are out of sync with master, where as this branch is in sync - so its showing more differences. will update the inner branches to match master, when siva resolves the fta pipeline issues :+1:

Reply:
Branches now in sync and this change has gone.